src/views/Admin/Modules/Records/List.vue

```diff
@@ -88,14 +89,20 @@ export default {
 
       return undefined
     },
+
+    moduleFields () {
+      return !!this.module.meta && !!Object.keys(this.module.meta).length > 0
```

Comment:
I think this could be written a bit better.

I'll spoil it for you 
```
    const { meta = { ui: {} } } = this.module || {}

    let fields = ((meta.ui || {}).admin || {}).fields || []
    fields = fields.length ? fields : this.module.fields
    ```